source/fundamentals/aggregation/aggregation-expression-operators.txt

```diff
@@ -0,0 +1,25 @@
+================================
+Aggregation Expression Operators
```

Comment:
I'm uncertain, but perhaps we might consider calling these "Aggregation Expression Operations" ("operator" has a certain incompatible meaning in Java that might be confusing).

Reply:
I think I prefer that too, though I'll need to rephrase part of the introductory paragraph to avoid confusion. Let me know what you think of the new version.

Reply:
Sounds good to me